website/docs/expansion.md

```diff
@@ -110,24 +54,78 @@ spec:
       kinds: ["Job"]
       versions: ["v1"]
   templateSource: "spec.template"
-  enforcementAction: "warn"
+  enforcementAction: "warn" # This will overwrite all constraint enforcement actions for the GVKs below that result from the GVKs above.
   generatedGVK:
     kind: "Pod"
     group: ""
     version: "v1"
 ```
 
 With this `ExpansionTemplate`, any constraints that are configured to target
-`Pods` will be evaluated on the "mock Pods" when a `Deployment` /`ReplicaSet` is
-being reviewed. Any violations created against the mock Pod will have their
+`Pods` will also be evaluated on the temporary, fake pods that Gatekeeper creates when a `Deployment` or `ReplicaSet` is
+being reviewed. Any violations created against these resulting `Pod`s, and only these resulting `Pod`s, will have their
 enforcement action set to `warn`, regardless of the enforcement actions
 specified by the Constraint in violation.
 
+If, for any reason, you want to exempt simulated resources from a specific
+constraint, look at the [Match Source](#match-source) section below.
```

Comment:
I think the main use case for using `Match.source` with ETs isn't to exempt specific resources (blacklist), but rather to only target the "fake" generated resources. 

In the case of mutators, one might be interested in simulating controller behavior by only mutating the "fake" resulting resources


In the case of constraints, one could create a Pod-reviewing constraint is specifically for validating expanded Pods. 

To this end, maybe something like:
"To see how to use Mutators and Constraints to exclusively apply to expanded resources, see the [Match Source](#match-source) section below." ?